kopf/logging/logging.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+import logging
+
+format = '[%(asctime)s] %(name)-20.20s [%(levelname)-8.8s] %(message)s'
+
+
+def configure(debug=None, verbose=None, quiet=None):
+    log_level = 'DEBUG' if debug or verbose else 'WARNING' if quiet else 'INFO'
+
+    logger = logging.getLogger()
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter(format)
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+    logger.setLevel(log_level)
+
+    # Configure the Kubernetes client defaults according to our settings.
+    try:
+        import kubernetes
+    except ImportError:
+        pass
+    else:
+        config = kubernetes.client.configuration.Configuration()
+        config.logger_format = format
+        config.logger_file = None  # once again after the constructor to re-apply the formatter
+        config.debug = debug
+        kubernetes.client.configuration.Configuration.set_default(config)
```

Comment:
This block already appears in `kopf/config.py`. I'm not sure how both relate. Could it be a duplicate?

Reply:
`kopf.logging` is a remnant of some early implementations, which has accidentally sneaked into the codebase when rebasing. It should not be here at all (and no other module use it) — so, removed.